File: exercicios/java/curso_programacao/src/Crescente.java
import java.util.Scanner;

public class Crescente {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int x, y;
		
		System.out.println("Digite dois numeros: ");
		x = sc.nextInt();
		y = sc.nextInt();
		
		while(x != y) {
			if(x < y) {
				System.out.println("CRESCENTE!");
			}else {
				System.out.println("DECRESCENTE!");
			}
			System.out.println("Digite outros dois numeros: ");
			x = sc.nextInt();
			y = sc.nextInt();
		}
		
		sc.close();
	}

}

File: exercicios/python/crescente.py
x: int
y: int

print('Digite dois numeros: ')

x = int(input())
y = int(input())

while x != y:
    if x < y:
        print('CRESCENTE!')
    else:
        print('DECRESCENTE!')
    print('Digite outros dois numeros: ')
    x = int(input())
    y = int(input())
